src/portalbackend/PortalBackend.DBAccess/Models/IdentityProvider.cs

```diff
@@ -0,0 +1,28 @@
+/********************************************************************************
+ * Copyright (c) 2022 BMW Group AG
+ * Copyright (c) 2022 Contributors to the Eclipse Foundation
```

Comment:
Please remove this line and change the copyright year to the current year.

```suggestion
 * Copyright (c) 2024 Contributors to the Eclipse Foundation
```